tests/gateway/test_braintree.py

```diff
@@ -263,250 +255,164 @@ def test_authorize_error_response(
     assert response.error == DEFAULT_ERROR
 
 
-@pytest.mark.integration
-@patch("saleor.payment.gateways.braintree.get_braintree_gateway")
-def test_authorize_incorrect_token(
-    mock_gateway, payment_dummy, braintree_not_found_error, gateway_config
-):
-    payment = payment_dummy
-    payment_token = "payment-token"
-    mock_response = Mock(side_effect=braintree_not_found_error)
-    mock_gateway.return_value = Mock(transaction=Mock(sale=mock_response))
-
-    payment_info = create_payment_information(payment, payment_token)
-    with pytest.raises(BraintreeException) as e:
-        authorize(payment_info, gateway_config)
-    assert str(e.value) == DEFAULT_ERROR_MESSAGE
+@pytest.fixture
+def sandbox_braintree_gateway_config(gateway_config):
+    """ set up your environment variables to record sandbox """
+    gateway_config.connection_params = {
+        "merchant_id": os.getenv("BRAINTREE_MERCHANT_ID", "9m6qhfxsqzm3cgzw"),
+        "public_key": os.getenv("BRAINTREE_PUBLIC_KEY", "fake_public_key"),
+        "private_key": os.getenv("BRAINTREE_PRIVATE_KEY", "fake_private_key"),
```

Comment:
I don't think that taking them from `env` is the best solution.  Someone can have these variables already set in his env and tests will fail or will generate strange behavior.

Reply:
I tried to prevent people (including myself) from committing credentials by accident but your scenario is troublesome indeed. I'll change it.